### DISCUSSION TITLE
[AG-Mobile] Situs NSFW Tidak Bisa Dibuka Melalui Google Images

### DISCUSSION THREAD

**Discussion by Jokopentil** [realodix/AdBlockID#1712] [Q&A]
https://www.google.com/search?q=bokep+pamer+jembut&tbm=isch&prmd=ivsnbmtz&dpr=2

**Comment by realodix** [comment]
Saya bingung dengan judul "Situs gambar tidak bisa di buka". Link yang Anda berikan link google, situs yang mana ya?

**Comment by Jokopentil** [comment]
Situs itu kan memuat gambar penelusuran Google
Nah setiap gambar itu ada link/bisa di buka situs sumber dari gambar itu
Nah itu linknya GK bisa ke buka

Pada Sen, 26 Feb 2024 00.40, Kei ***@***.***> menulis:

**Comment by Jokopentil** [comment]
Saya mau kasih screenshot jadi ragu karena gambarnya dewasa

**Comment by realodix** [comment]
> Situs itu kan memuat gambar penelusuran Google
> Nah setiap gambar itu ada link/bisa di buka situs sumber dari gambar itu
> Nah itu linknya GK bisa ke buka

Saya masih belum ngerti tentang gak bisa dibuka tuh gimana? Saya buka link, biasa aja, gak ada masalah.

> Saya mau kasih screenshot jadi ragu karena gambarnya dewasa

Upload saja di https://imgur.com/, lalu taruh linknya di sini

**Reply by Jokopentil** [reply]
Gini loh apakah kalo di buka di PC bisa di buka
Saya membukanya menggunakan hp
Itu itu situs internet positif tapi dianggap pop up oleh adguard
Kadang bisa di buka tapi setelah itu GK bisa sama sekali aku upload di sini saja karena cukup ribet jika harus bikin akun
Tapi saya blur saja
Link yg saya lingkari ini GK bisa di buka
Browser pakai kiwi
Saya belum mencobanya pakai Firefox 
![IMG_20240226_105246.jpg](https://github.com/realodix/AdBlockID/assets/114223791/aa92624b-5b2d-43f8-a165-e78df9540ad5)

![IMG_20240226_105133.jpg](https://github.com/realodix/AdBlockID/assets/114223791/fa302a44-f5f7-48cc-be6e-4e500d2ec6e3)
